locales/en/plugin__odf-console.json

```
@@ -1110,8 +1110,8 @@
   "Client onboarding token": "Client onboarding token",
   "Copy to clipboard": "Copy to clipboard",
   "How to use this token?": "How to use this token?",
-  "To onboard the client cluster, the provider cluster requires the onboarding token.": "To onboard the client cluster, the provider cluster requires the onboarding token.",
-  "An onboarding token is needed to connect an additional OpenShift cluster to a Data Foundation deployment.": "An onboarding token is needed to connect an additional OpenShift cluster to a Data Foundation deployment.",
+  "An onboarding token is needed to connect an additional OpenShift cluster to an OpenShift Data Foundation deployment. Copy the generated token and use it for deploying OpenShift Data Foundation client operator on your OpenShift cluster.": "An onboarding token is needed to connect an additional OpenShift cluster to an OpenShift Data Foundation deployment. Copy the generated token and use it for deploying OpenShift Data Foundation client operator on your OpenShift cluster.",
```

cc @Chandan141996 (no "Openshift Data Foundation", only "Data Foundation")...
```suggestion
  "An onboarding token is needed to connect an additional OpenShift cluster to an Data Foundation deployment. Copy the generated token and use it for deploying Data Foundation client operator on your OpenShift cluster.": "An onboarding token is needed to connect an additional OpenShift cluster to an OpenShift Data Foundation deployment. Copy the generated token and use it for deploying OpenShift Data Foundation client operator on your OpenShift cluster.",
```

---

Done